packages/python-neo4j/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-neo4j
+_pkgname=neo4j
+pkgver=4.4.5
+pkgrel=1
+pkgdesc='A Neo4j driver for Python.'
+arch=('any')
+url='https://pypi.org/project/neo4j/'
+license=('Apache2')
+depends=('python' 'python-pytz')
```

Comment:
shouldn't it requires neo4j then? if it's a python wrapper for neo4j 

package: [neo4j-community](https://github.com/BlackArch/blackarch/blob/master/packages/neo4j-community/PKGBUILD)

Reply:
also, isn't it the `python-neo4j-driver` we already offer? i didn't check further. please check if it's the same package.

Reply:
yeah turns out that they are the same package. thanks for pointing that out @noptrix